### ISSUE TITLE
Regression in v3.12.0 - Cannot resolve a Slate point from DOM point: [object Text],0

### ISSUE THREAD

**Issue by SiddharthSham** [sanity-io/sanity#4576]
**Describe the bug**

On `v3.12.0`, focusing the cursor in any Portable Text Editor field throws the following error:
```
Cannot resolve a Slate point from DOM point: [object Text],0
```
This used to work perfectly in `v3.11.5`, so it must be a regression in `v3.12.0`

**To Reproduce**

1. Ensure you are using version 3.12.0
2. Create a Portable Editor Field
3. Attempt to type in any content / focus the cursor

**Expected behavior**
Content can be input

**What operating system are you using?**
MacOS Ventura 13.4 (22F66)

**Which versions of Node.js / npm are you running?**
v18.14.2

**Additional context**
The problem seems to originate from Slate, so it might be a quickfix to rollback to a working version.

**Comment by bobinska-dev** [comment]
Hello @SiddharthSham 👋 
Could you update your sanity versions and see if this error persists? We are not able to recreate this....

**Comment by SiddharthSham** [comment]
Hello @bobinska-dev , I. did test this last weekend with `v3.14.1` as well, and I confirm the error still exists with PortableText :(

**Comment by umxr** [comment]
@SiddharthSham Are you getting this error with a custom input? or just the native portable text block. im having the same issue. in my case, we have a custom input

**Comment by SiddharthSham** [comment]
@umxr I get the error regardless of whether a custom input is being used or not. However, I did first notice it when using a custom input (for field-level localisation). Are you able to get rid of this error by removing the custom error, by any chance?

**Comment by d4v1d82** [comment]
Sorry @umxr wrong copy/paste, I've updated the comment with the right version that is `3.11.5`

**Comment by SimeonGriggs** [comment]
This error, when seen in the Internationalized Array plugin, is fixed in version [1.10.0](https://github.com/sanity-io/sanity-plugin-internationalized-array/releases/tag/v1.10.0)
